src/DIRAC/Resources/Storage/test/FIXME_Test_StorageElement.py

```diff
@@ -61,21 +61,21 @@ def test_isValid(self):
         res = self.storageElement.isValid()
```

Comment:
This test is in "FIXME", but it seems it might become useful for this occasion.

Reply:
What I wrote in the `pytest` file is pretty much equivalent of that. I may even remove that file